README.md

```diff
@@ -4,56 +4,140 @@
 
-## Local installation
-- Clone the repository:
-  ```
+## Local installation - 
+  ### Clone the repository :
+  ```bash
   git clone https://github.com/neurobagel/query-tool-ai.git
   cd query-tool-ai
   ```
-- Create and activate a virtual environment:
-   ```
+
+  After cloning the repository, you can choose to either use the Dockerized version or run the application locally using Python. 
+  #### Follow the instructions for the option that suits you best:
+
+  ### Option 1 : Docker :
+  - #### First, [install Docker](https://docs.docker.com/get-docker/).
+  - #### Build the image using the Dockerfile
+    After cloning the current repository, build the Docker image locally:
+    ```bash
+    docker build -t neurobagel-query-tool-ai .
+    ```
+  - #### Run the container from the built image:
+    Start the container from the built image. Ensure you map the necessary volumes and set the container name.
+    ```bash
+    docker run -d \
+    -p 8000:8000 \
+    -v ollama:/root/.ollama \
+    -v /home/user/data:/app/output/ \
+    --name query-tool-ai-container neurobagel-query-tool-ai
+    ```
+
+    Then you can access the query tool ai at http://localhost:8000
+
+    **Note:** the query tool ai is listening on port 8000 inside the docker container, replace port 8000 by the port you would like to expose to the host. For example if you'd like to run the tool on port 8080 of your machine you can run the following command:
+    ```bash
+    docker run -d \
+    -p 8080:8000 \
+    -v ollama:/root/.ollama \
+    -v /home/user/data:/app/output/ \
+    --name query-tool-ai-container neurobagel-query-tool-ai
+    ```
+
+
+  - #### Verify Host and Port
+    To ensure the server is running on localhost, you can check the logs or try accessing the server in your browser or with `curl`(assuming the the query tool ai is listening on port 8000):
+    ```bash
+    curl -X GET "http://localhost:8000/"
+    ```
+    You should see the response:
+    ```json
+    {"message": "Welcome to the Neurobagel Query Tool AI API"}
+    ``` 
+
+### Option 2 : Python
+- #### Create and activate a virtual environment:
+   ```bash
    python3 -m venv venv
    source venv/bin/activate
    ```
-- Install the required packages and pull the Ollama model:
-   ```
+- #### Install the required packages and pull the Ollama model:
+   ```bash
    pip install -r requirements.txt
    ollama pull mistral
    ```
-- Run the python script:
-  ```
-  python3 app/api/url_generator.py
-  ```
-- You should see the following prompt:
-  ```
-  Enter user query (or 'exit' to quit): 
-  ```
-  Enter your query to get the desired API URL.
 
-## Dockerized version
-- First, [install Docker](https://docs.docker.com/get-docker/).
+-  #### Set the environment variables.
```

Comment:
I think it's better to have this at the top of the `local installation` section after `Clone the repository` sub-section since it applies to both options.